rcl/src/rcl/node.c

```diff
@@ -179,7 +211,47 @@ rcl_node_init(
   }
   // actual domain id
   node->impl->actual_domain_id = domain_id;
-  node->impl->rmw_node_handle = rmw_create_node(name, local_namespace_, domain_id);
+
+  const char * ros_enable_security = NULL;
+  const char * ros_enforce_security = NULL;
+
+  if (rcutils_get_env(sros_enable_security_var_name, &ros_enable_security)) {
+    RCL_SET_ERROR_MSG("fail fetching environment variable", rcl_get_default_allocator());
```

Comment:
That would be great indeed, I don't remember RCL_SET_ERROR_MSG taking formatting arguments so I didn't do it. I could write directly the variable name in the string for each condition but that seems wrong to me so I stuck with just saying that it failed. I can replace 'fetching' with 'to read' though